CHANGELOG.md

```diff
@@ -41,18 +44,18 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Deprecated
 - The argument `_py2_no_more_posargs` because Python2 is removed on January
-, 2020 ([#328](https://github.com/pdfminer/pdfminer.six/pull/328) and 
+, 2020 ([#328](https://github.com/pdfminer/pdfminer.six/pull/328) and
```

Comment:
Did you change these lines on purpose?

Reply:
Nope, that's my editor removing whitespace from the ends of lines... will edit that commit.